Can not find python3

Hello again,

I have a python3 codebase with pip dependencies.

When I run

```
docker/run.sh "-v /my-project:/project" --info analyze -f JSON -i /project -o /project/ort/analyzer
```

I get back a json with:
```
        "PIP::requirements.txt:" : [ {
          "timestamp" : "2019-07-12T16:05:36.880494Z",
          "source" : "PIP",
          "message" : "IOException: Cannot run program \"python3\": error=2, No such file or directory\nCaused by: IOException: error=2, No such file or directory",
          "severity" : "ERROR"
        } ]
```

Best,
Felix

--- a/docker/run/Dockerfile
+++ b/docker/run/Dockerfile
@@ -63,6 +63,7 @@ RUN apt-get update && \
         haskell-stack=$HASKELL_STACK_VERSION \
         npm=$NPM_VERSION \
         python-pip=$PYTHON_PIP_VERSION \
+        python3-pip=$PYTHON_PIP_VERSION \
         sbt=$SBT_VERSION \
     && \
     rm -rf /var/lib/apt/lists/*

